doc/userguide/capture-hardware/myricom.rst

```diff
@@ -23,7 +23,7 @@ Compile Suricata against Myricom’s libpcap:
   make
   sudo make install
 
-Next, configure the amount of ringbuffers. I’m going to work with 8 here, as my quad core + hyper threading has 8 logical CPU’s. *See below* for additional information about the buffer-size parameter.
+Next, configure the amount of ringbuffers. I'm going to work with 8 here, as my quad core + hyper threading has 8 logical CPU's. *See below* for additional information about the buffer-size parameter.
```

Comment:
Not sure if it matters but it should be `CPUs`. Apostrophe should not be used here at all.